apis/kubedb/v1alpha2/conversion.go

```diff
@@ -622,10 +622,19 @@ func Convert_v1_MongoDBSpec_To_v1alpha2_MongoDBSpec(in *v1.MongoDBSpec, out *Mon
 	return nil
 }
 
+func Convert_v1alpha2_RedisClusterSpec_To_v1_RedisClusterSpec(in *RedisClusterSpec, out *v1.RedisClusterSpec, s conversion.Scope) error {
+	out.Shards = (*int32)(unsafe.Pointer(in.Master))
+	out.Replicas = (*int32)(unsafe.Pointer(in.Replicas))
```

Comment:
out.Replicas = in.Replicas + 2

// Replica count should tell how many nodes in the shard including master and replica

Reply:
out.Replicas = in.Replicas + 1 ?

Reply:
@AbdullahAlShaad

Reply:
Yes